subsys/logging/log_core.c

```diff
@@ -233,6 +234,7 @@ static void z_log_msg_post_finalize(void)
 			k_sem_give(&log_process_thread_sem);
 		}
 	} else {
+		; /* Non-empty final else in if ... else if struct */
```

Comment:
`No action needed. Message processing will be triggered by the timeout or when number of upcoming messages exceeds  the threshold.`